app/classifier/task-nav.jsx

```diff
@@ -124,7 +125,7 @@ class TaskNav extends React.Component {
 
     const disableTalk = this.props.classification.metadata.subject_flagged ||
     (isFeedbackActive(this.props.project) && isThereFeedback(this.props.subject, this.props.workflow));
-    const visibleTasks = Object.keys(this.props.workflow.tasks).filter((key) => { return this.props.workflow.tasks[key].type !== 'shortcut'; });
+    const visibleTasks = Object.keys(this.props.workflow.tasks).filter(key => this.props.workflow.tasks[key].type !== 'shortcut');
```

Comment:
I don't think the changes on lines 13-18 and here will break anything, but commenting to give ourselves a heads up on the script changing these from explicit returns to implicit.